docs/requirements-rtd.txt

```diff
@@ -8,10 +8,10 @@ setuptools_scm
 sphinx<7.0.0
 sphinx_rtd_theme
 funcparserlib==1.0.0a0
-sphinxcontrib-seqdiag
 numpydoc
 pytest
 PyGObject
 dbus-next
 cairocffi>=1.6.0
-xcffib>=1.4.0
\ No newline at end of file
+xcffib>=1.4.0
+libcst
```

Comment:
any minimum version needed? Maybe we can just set it to >= 1.0.0 or not?

Reply:
Good idea.